makejdk.sh

```diff
@@ -75,6 +75,9 @@ while [[ $# -gt 0 ]] && [[ ."$1" = .-* ]] ; do
 
     "--jtreg" | "-j" )
     JTREG=true; shift;;
+    
+    "--jtreg_subsets" )
+    JTREG=true; JTREG_TEST_SUBSETS="$1"; shift;;
```

Comment:
karianna suggests that we're making a mountain out of a molehill here, so he proposed (and I agreed) that we should add the functionality now and tidy it up later when we have spare time.